src/main/java/com/extremelyd1/game/progress/ProgressController.java

```diff
@@ -0,0 +1,45 @@
+package com.extremelyd1.game.progress;
+
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.util.List;
+
+public class ProgressController {
+
+    /**
+     * Rules for lines notifications
+     */
+    private List<String> linesRules;

```

Comment:
This (and all other member variables in this class) can be `final`.

Reply:
done